### PR TITLE
Improve/paloalto

### PR DESCRIPTION
**Pull request by TOUFIKIzakarya**
related issue : [#48988](https://github.com/SekoiaLab/platform/issues/48988)

### CODE REVIEW

File: Palo Alto Networks/paloalto-ngfw/ingest/parser.yml [hunk 1]
```diff
@@ -75,6 +75,120 @@ pipeline:
           - pkts_received
         delimiter: ","
 
+  # TRAFFIC CSV with hight Resolution timestamp
```
Comment (squioc) [inline Palo Alto Networks/paloalto-ngfw/ingest/parser.yml:78]: I think it's preferable to extend the previous stage instead to create a new one.
I believe that the `dsv.parse-dsv` is able to manage when records haven't the same number of columns defined in the columnnames property. Can you check that?

File: Palo Alto Networks/paloalto-ngfw/ingest/parser.yml [hunk 2]
```diff
@@ -490,6 +604,10 @@ stages:
           rule.uuid: "{{parsed_event.message.PanOSRuleUUID or parsed_event.message.RuleUUID}}"
           source.bytes: "{{parsed_event.message.BytesSent or parsed_event.message.in}}"
 
+      - set:
+          event.timezone: "{{'+' + parsed_event.message.HighResolutionTimestamp.split('+')[1]}}"
+        filter: "{{parsed_event.message.HighResolutionTimestamp != None}}"
+
```
Comment (squioc) [inline Palo Alto Networks/paloalto-ngfw/ingest/parser.yml:610]: `HighResolutionTimestamp` must be used to set `@timestamp`. Can you use it in a date.parse stage? Thanks

**Review by squioc — CHANGES_REQUESTED** [review]

File: Palo Alto Networks/paloalto-ngfw/tests/fix_bug_without_int.json [hunk 3]
```diff
@@ -41,14 +41,19 @@
...
+      "hostname": "FWPA01",
       "logger": "traffic"
     },
     "network": {
       "application": "windows-remote-management",
       "bytes": 2346,
       "packets": 9,
-      "transport": "tcp"
+      "transport": "tcp",
+      "type": "N/A"
```
Comment (squioc) [inline Palo Alto Networks/paloalto-ngfw/tests/fix_bug_without_int.json:56]: Can you add `N/A` in the `ignored_values` of the parser?

Comment (TOUFIKIzakarya) [reply Palo Alto Networks/paloalto-ngfw/tests/fix_bug_without_int.json]: Thank you, I didn't see it.

**Review by squioc — DISMISSED** [review]

**Review by squioc — APPROVED** [review]

**Pull request merged**